library/Zend/Pdf/Canvas.php

```diff
@@ -77,7 +77,7 @@ public function __construct($width, $height)
      */
     protected function _addProcSet($procSetName)
     {
-        $this->_procset[$procSetName] = 1;
+        $this->_procSet[$procSetName] = 1;
```

Comment:
match commit messages with actual changes. you are not adding the property here, but rather fix typo.

Reply:
Thanks Mr @glensc i changed commit message here https://github.com/Shardj/zf1-future/pull/291/commits/6c35ae87d85d70b1a1f2a906afb58fd9cee0a382